apps/blockchain/lib/blockchain/block.ex

```diff
@@ -23,18 +23,21 @@ defmodule Blockchain.Block do
   # header: B_H,
   # transactions: B_T,
   # ommers: B_U
+  # additonal_info: precomputated data required by JSON RPC spec
   defstruct block_hash: nil,
             header: %Header{},
             transactions: [],
             receipts: [],
-            ommers: []
+            ommers: [],
+            additional_info: nil
```

Comment:
would `metadata` be a better name for this? If not, is there anything more specific than `additional_info` that would fit here?